.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Feature request
```

Comment:
Since MAAS feature requests are tracked elsewhere-- is there any consensus on what to do with ansible playbook feature requests?